pom.xml

```diff
@@ -299,47 +299,47 @@
       <plugins>
         <plugin>
           <artifactId>maven-clean-plugin</artifactId>
-          <version>2.6</version>
+          <version>3.0.0</version>
         </plugin>
         <plugin>
           <artifactId>maven-compiler-plugin</artifactId>
-          <version>3.3</version>
+          <version>3.6.1</version>
```

Comment:
This is the compiler plugin version used by the git plugin and the git client plugin and it has worked well for me.